packages/velaux-ui/src/pages/Login/index.tsx

```diff
@@ -133,107 +156,116 @@ export default class LoginPage extends Component<Props, State> {
-                  <div className="logo-img-wrapper">
-                    <img src={Logo} />
-                  </div>
-                  <h3 className="login-title-description">
-                    <Translation>Make shipping applications more enjoyable</Translation>
-                  </h3>
-                  <Form
-                    onSubmitCapture={(e) => {
-                      e.preventDefault();
-                    }}
-                    onSubmit={this.handleSubmit}
-                    {...formItemLayout}
-                    field={this.field}
-                  >
-                    <FormItem
-                      label={<Translation className="label-title">Username</Translation>}
-                      labelAlign="top"
-                      required
-                    >
-                      <Input
-                        name="username"
-                        placeholder={i18n.t('Please input the username').toString()}
-                        {...init('username', {
-                          rules: [
-                            {
-                              required: true,
-                              pattern: checkName,
-                              message: <Translation>Please input the username</Translation>,
-                            },
-                          ],
-                        })}
-                      />
-                    </FormItem>
-                    <FormItem
-                      label={<Translation className="label-title">Password</Translation>}
-                      labelAlign="top"
-                      required
-                    >
-                      <Input
-                        name="password"
-                        htmlType="password"
-                        placeholder={i18n.t('Please input the password').toString()}
-                        {...init('password', {
-                          rules: [
-                            {
-                              required: true,
-                              pattern: checkUserPassword,
-                              message: (
-                                <Translation>
-                                  The password should be 8-16 bits and contain at least one number and one letter
-                                </Translation>
-                              ),
-                            },
-                          ],
-                        })}
-                      />
-                    </FormItem>
-                    <Button loading={loginLoading} type="primary" htmlType="submit" onClick={this.handleSubmit}>
-                      <Translation>Sign in</Translation>
-                    </Button>
-                  </Form>
-                  <If condition={loginErrorMessage}>
-                    <div className="logo-error-wrapper">
-                      <Icon type="warning1" /> <Translation>{loginErrorMessage}</Translation>
+          <If condition={!firstTimeLogin}>
```

Comment:
How do you format your code? This style is different from the default.

Reply:
I have adjusted my IDE Typescript code format to align with default. GitHub recognize the deletion and creation because I add a `<If>` so lot's of indent appear.